_posts/2022-02-01-Midi-Mixerboard-Control.md

```diff
@@ -0,0 +1,139 @@
+3. Install ASIO4ALL (if you do not have an audio device with an ASIO driver).
+
+### Setting everything up
+*Plug in your MIDI Control Surface into your computer. This needs to be done before you start QjackCtl and Jamulus.*
+
+Now let's setup JACK:
+
+You will mostly control the Audio flow in JACK (or QjackCtl, the software you can see in the screenshot which you can find in the Start Menu).
+
+![QjackCtl-start](https://user-images.githubusercontent.com/13550012/152056827-8e0c12e4-a683-49ee-a7a5-f3ce3d6d3e2f.png)
+
+
+Click the **Setup** button in the QjackCtl software to select your Audio interface or ASIO4ALL.
+You should use/select an ASIO driver:
+
+![QjackCtl-setup](https://user-images.githubusercontent.com/13550012/152056869-fbab591c-5271-4502-9efc-1670ad8ce720.png)
+
+Ensure the driver is set to portaudio, the Sample Rate is set to 48000,  Frames/Period is set to something low like 64 or 128. You can now close this window by clicking **OK**.
+
+Next, click on the **Start** button in Qjackctl.
+
+![QjackCtl-started](https://user-images.githubusercontent.com/13550012/152056960-8fc51bc6-5cc0-492e-9ff1-298a8dff4c74.png)
+
+After this is started, click the **Graph** button to show the Graph dialog:
+
+![QjackCtl-at-startup](https://user-images.githubusercontent.com/13550012/152056935-b133a172-9894-4a2b-96dc-5aa7e4c5407b.png)
+
+*NOTE: In this dialog you should see the System MIDI connection. If you do not see that option, then your MIDI device is not recognized or you may have connected the device after you started JACK.*
+
+Now let's update the Jamulus shortcut:
+
+The Jamulus installation has created a Desktop Shortcut, which we need to modify. We need to tell Jamulus that we want to use the MIDI Control Surface.
+Locate the icon on the desktop, right click and select "Properties"
+
+![jamulus-desktop-shortcut-properties](https://user-images.githubusercontent.com/13550012/152057006-b1878219-e661-4548-9bd3-0fa36422ddb0.png)
+
+We need to add an option to the Target field.
+
+The information you need to provide will depend on the MIDI Control Surface device configuration.
+The mixing section of my device contains 9 faders, 9 knobs and 9 buttons. As I only have 1 button row, I need to decide to use the Mute or Solo option.
+- The fader section starts with CC# 3 and ends with CC# 11.
+- The knobs section starts with CC# 14 and ends with CC# 22.
+- The button sections starts with CC# 23 and ends with CC# 31.
+
+We need to tell Jamulus that these Control Change numbers are being used. When the Control Change is sent, it will also sent a value for the CC. The value will be in the range from 0 till 127.
+The fader at the zero position will sent value 0 and at the max. position it will sent value 127.
+The knob rotate and will sent 0 when turned fully to the left, 64 when in top centre and 127 when turned fully to the right.
+The button will sent OFF (0 value) or ON (127 value).
```

Comment:
send

Reply:
Seems to be a theme.....with incorrect usage of sent :)